modules/apps/frontend-data-set/frontend-data-set-admin-web/src/main/resources/META-INF/resources/js/data_set/actions/components/ActionForm.tsx

```diff
@@ -899,7 +885,16 @@ const ActionForm = ({
 			)}
 
 			<ClayButton.Group className="pb-4 px-4" spaced>
-				<ClayButton disabled={saveButtonDisabled} onClick={saveAction}>
+				<ClayButton
+					disabled={saveButtonDisabled}
+					onClick={() => {
+						const valid = validate();
+
+						if (valid) {
+							saveAction();
+						}
+					}}
+				>
```

Comment:
Implementing standard validation behavior:
- Button is always enabled, related to validation. It is disabled only during saving.
- On button click, we do full form validation.
- There are also individual validations of form elements, in addition to full form validation.